from exceptions import divide

divide(4, 3)